spec/controllers/app_configs_controller_spec.rb

```diff
@@ -146,4 +146,68 @@
       it { is_expected.to render_template('application_setup/index') }
     end
   end
+
+  describe '#run_daily_tasks (PUT /app_configs/run_daily_tasks)' do
+    before(:each) do
+      mock_app_config
+      request.env['HTTP_REFERER'] = 'where_i_came_from'
+    end
+
+    context 'as superuser' do
+      before(:each) do
+        sign_in FactoryGirl.create(:superuser)
+      end
+
+      it 'enqueues the daily tasks' do
+        expect(DailyTasksJob).to receive(:perform_now)
```

Comment:
Should we be testing `HourlyTasksJob` and `DailyTasksJob` just to make sure that they, in turn, call the correct jobs? I feel like that's worth having, especially since it would all be stubbed anyway.